cmd/cli/docs/botkube_uninstall.md

```diff
@@ -0,0 +1,52 @@
+---
+title: botkube uninstall
+---
+
+## botkube uninstall
+
+uninstall Botkube from cluster
+
+### Synopsis
+
+Use this command to uninstall the Botkube agent.
+
+```
+botkube uninstall [OPTIONS] [flags]
+```
+
+### Examples
+
+```
+# Uninstall default Botkube Helm release
+botkube uninstall
+
+# Uninstall specific Botkube Helm release
+botkube uninstall --release-name botkube-dev
+```
+
+### Options
```

Comment:
It'd be great to add another flag which skips the confirmation. e.g. `-y`? That will be great for CI pipelines.

Reply:
I added `-y, --auto-approve` flag both to install and uninstall 👌